templates/etc_systemd_system_postgresql.service.d_custom.conf.j2

```diff
@@ -8,4 +8,8 @@ Group={{ postgresql_service_group }}
 Environment=PGDATA={{ postgresql_conf_directory }}
 
 ExecStartPre=
+    {% if postgresql_version == 10 %}
```

Comment:
At this point I'd say `>= 10`, as I suspect this format will stay for v11, v12, etc.

Reply:
Done.

Reply:
Sorry for re-reviewing... but we should be using the `version_compare()` function here. (and anywhere else you're comparing versions)

` {% if postgresql_version | version_compare ('10', '>=') %}`

Reply:
Done ;-)